packages/nc-gui-v2/components/smartsheet-header/Menu.vue

```diff
@@ -1,10 +1,54 @@
 <script lang="ts" setup>
+import { Modal } from 'ant-design-vue'
+import { inject } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useToast } from 'vue-toastification'
+import { useNuxtApp } from '#app'
+import { useMetas } from '#imports'
+import { ColumnInj, MetaInj } from '~/context'
+import { extractSdkResponseErrorMsg } from '~/utils/errorUtils'
 import MdiEditIcon from '~icons/mdi/pencil'
 import MdiStarIcon from '~icons/mdi/star'
 import MdiDeleteIcon from '~icons/mdi/delete-outline'
 import MdiMenuDownIcon from '~icons/mdi/menu-down'
 
+const { virtual = false } = defineProps<{ virtual?: boolean }>()
 const editColumnDropdown = $ref(false)
+
+const column = inject(ColumnInj)
+const meta = inject(MetaInj)
+
+const { $api } = useNuxtApp()
+const { t } = useI18n()
+const toast = useToast()
+const { getMeta } = useMetas()
+
+const deleteColumn = () =>
+  Modal.confirm({
+    title: h('div', { innerHTML: `Do you want to delete <span class="font-weight-bold">'${column?.title}'</span> column ?` }),
```

Comment:
instead of `innerHTML` use the 3rg argument for `h` and pass the string as a slot.